src/SdkCommon/ClientRuntime.Azure/ClientRuntime.Azure/JsonSerializerExtensions.cs

```diff
@@ -25,7 +25,10 @@ public static class JsonSerializerExtensions
             }
             JsonSerializer newSerializer = new JsonSerializer();
             var properties = typeof(JsonSerializer).GetTypeInfo().DeclaredProperties;
-            foreach (var property in properties.Where(p => p.SetMethod != null && !p.SetMethod.IsPrivate))
+            foreach (var property in properties.Where(p => 
```

Comment:
@hovsepm what if Binder property is being set by the serializingSettings?
Will be ignore in that case.
I think we should not ignore Obsolete properties, rather ignore properties that are throwing exceptions.